README.md

```diff
@@ -0,0 +1,7 @@
+#Styles structure
+
+*index.css - compiled style file, it's mounted once in root index.tsx
+*index.scss - root file to mount all .scss files
+*app.scss - variables (breakpoints, color, etc.), mixins
```

Comment:
Regarding app.scss, maybe lets use more intuitive name, e.g. variables.scss? It's just a assumption.